cmd/defaults.go

```diff
@@ -8,4 +8,5 @@ var (
 	DefaultNamespace         = "default"
 	DefaultServiceAccount    = "default"
 	DefaultScorecardWaitTime = "240"
+	DefaultPyxisHost         = "pyxis.engineering.redhat.com"
```

Comment:
I don't think this URL is accessible without SSO or VPN. I think we want to use `catalog.redhat.com` for this.